samples/subsys/senss/src/main.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <senss/senss.h>
+#include <stdio.h>
+
+static int data_event_callback(
+		senss_sensor_handle_t handle,
+		void *buf, int size, void *param)
+{
+	struct senss_sensor_info info;
+	senss_get_sensor_info(handle, &info);
+
+	printf("Received sensor data from sensor: %s, type: %d, instance: %d\n",
+		info.name,
+		info.type,
+		info.instance
+	);
+
+	if (info.type == SENSS_SENSOR_TYPE_MOTION_ACCELEROMETER_3D) {
+		struct senss_sensor_value_3d_int32* value = buf;
```

Comment:
Perhaps my comments about not supporting arbitrary channels was premature, at least this part of the interface seems pretty extensible

Reply:
Yes,  the multiple clients and arbitration supports are core functions in `senss`.